io.github.mgerhardy.vengi.voxedit.yml

```diff
@@ -49,8 +50,10 @@ modules:
     sources:
       - type: git
         url: https://github.com/mgerhardy/vengi
```

Comment:
this url has changed to https://github.com/vengi-voxel/vengi

Reply:
If you also want to change ID, you have to resubmit at https://github.com/flathub/flathub/ and deprecate this repository.